pwiz_tools/Shared/zedgraph/ZedGraph/GraphPane.cs

```diff
@@ -1549,39 +1559,64 @@ public void AdjustLabelSpacings(List<LabeledPoint> labPoints, Control parentCont
                     }
                     else
                         labeledPoint.Label.IsVisible = false;
-                }
-
+                }
+				
                 if (visiblePoints.Any())
                 {
-                    GraphObjList.RemoveAll(obj => obj is BoxObj || obj is LineObj);
+                    var savedLayout = existingLayout ?? new List<LabeledPoint.PointLayout>();
+					var newPoints = new List<LabeledPoint>();
                     foreach (var point in visiblePoints)
                     {
-                        if (_labelLayout.PlaceLabel(point, g))
+                        // if we already have a saved point layout for this peptide we do not need to re-adjust it
+						// CONSIDER: There is no stable persistent protein ID, so we cannot match the points using identityPath
```

Comment:
In theory, you can get a persistent ID from an IdentityPath by doing:
`new ElementRefs(srmDocument).GetNodeRef(identityPath)`